admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
@intoxstudio Maybe change this to `delete_users` since that is the cap that is checked for `is_super_admin()`. Let me know what you think!